.github/workflows/binaries.yml

```diff
@@ -0,0 +1,55 @@
+on:
+  release:
+    types: [created]
+name: Binary artifacts
+jobs:
+  generate:
+    env:
+      # The version needs to match with the one used in the printer
+      PRETTIER_API: 2.0.0
```

Comment:
I'm not convinced by the API terminology. Can we use `PRETTIER_CORE_LIB_VERSION` instead or something more explicit